### DISCUSSION TITLE
Toxic Spore nodes markers

### DISCUSSION THREAD

**Discussion by Void2258** [blish-hud/Blish-HUD#701] [Requested Marker Packs]
A marker set for Toxic Spore spawn locations in Kessex Hills would be very useful. The locations of toxic offshoot events would also be helpful, but as they are not player triggerable they are less farmable.

Here is an old map marking the spawn spots, but it still seems to be accurate: [map](https://i.imgur.com/kY9yZi6.png)

**Comment by Metallis** [comment]
I've taken an initial pass at this using the image provided. Only 2 locations on the map even spawned and I believe only 1 matched. Does this image look more detailed? https://wiki.guildwars2.com/wiki/Toxic_Seedling#/media/File:Toxic_Seedling_map.jpg

**Comment by Void2258** [comment]
These are maps of POSSIBLE spawn locations. As noted, only 6 can be active on a given map instance. I do not know which of these are more accurate maps, though the one on the wiki should be more up to date (the other one predates the changes to progression system).

**Reply by Metallis** [reply]
It was assumed this was asking for all possible spawn locations, I just wanted to confirm I was marking the correct thing. I'll look into the Offshoot event locations here shortly.

**Comment by Metallis** [comment]
This has been completed and integrated into https://github.com/Metallis/Metal-Marker-Myriad
